.github/dependabot.yml

```diff
@@ -9,4 +9,4 @@ updates:
   - package-ecosystem: "composer"
     directory: "/"
     schedule:
-      interval: "monthly"
+      interval: "monthly"
```

Comment:
The warning sign indicates that you lack a final newline, which is a common convention in particular in line-based systems such as git.
![image](https://github.com/user-attachments/assets/4142eb93-c884-4abe-9451-ba72ef8880c2)
If you enable https://editorconfig.org/ in your favourite IDE, the rules of `.editorconfig` will be applied and automatically avoid this when you save a file.